packages/twenty-front/src/generated/graphql.tsx

```diff
@@ -3884,7 +3884,12 @@ export type GetEnvironmentVariablesGroupedLazyQueryHookResult = ReturnType<typeo
 export type GetEnvironmentVariablesGroupedQueryResult = Apollo.QueryResult<GetEnvironmentVariablesGroupedQuery, GetEnvironmentVariablesGroupedQueryVariables>;
 export const UpdateLabPublicFeatureFlagDocument = gql`
     mutation UpdateLabPublicFeatureFlag($input: UpdateLabPublicFeatureFlagInput!) {
-  updateLabPublicFeatureFlag(input: $input)
+  updateLabPublicFeatureFlag(input: $input) {
+    id
+    key
+    value
+    workspaceId
```

Comment:
I'm surprised we need the workspaceId since it's always for my current workspace ; seems a bit misleading to have it available in the backend